FirebaseAuth/Tests/Unit/FIRAuthTests.m

```diff
@@ -1374,6 +1421,54 @@ - (void)testSignInWithGoogleCredentialFailure {
   OCMVerifyAll(_mockBackend);
 }
 
+/** @fn testSignInWithAppleCredentialSuccess
+    @brief Tests the flow of a successful @c signInWithCredential:completion: call
+        with an Apple Sign-In credential.
+ */
+- (void)testSignInWithAppleCredentialSuccess {
```

Comment:
nit: Can you name this `testSignInWithOAuthCredentialWithDisplayNameSuccess` to make it more clear the purpose of this test?